backend/app/notify/telegram_test.go

```diff
@@ -496,3 +496,13 @@ func mockTelegramServer(h http.HandlerFunc) *httptest.Server {
 
 	return httptest.NewServer(router)
 }
+
+func TestTelegram_telegramSupportedHTML(t *testing.T) {
```

Comment:
I propose to test it next to `// test buildMessage separately for message text`, that way it would be less painful to change the implementation later on but verify that the logic stays the same.

Ideally, we need to test these things by calling `Telegram.Send` and don't care about how it gets generated internally, but we don't have a mock server setup in the tests yet to do it.  There are fewer steps between `buildMessage` and `Send` than between the new function and it, so testing `buildMessage` will simplify including this test case for whoever will rewrite tests later on.